src/kOS/Control/SteeringManager.cs

```diff
@@ -1319,7 +1319,7 @@ object IFlightControlParameter.GetValue()
         {
             return Value;
         }
-            
+
```

Comment:
For whatever reason, Monodevelop inserts space into the file for indenting purposes *regardless* of whether you actually type anything on that line or not.  It can be frustrating to have to constantly have to manually re-edit it back out.

Reply:
Yeah, I thought about not making the edit.  It just had stood out to me in the diff because it was not surrounded by changes.  I suspect they lack the logic that is used in visual studio, where it will indent your cursor but only add the spaces if you actually type at that indent.